etc/systemd/system/zfs-import-cache.service.in

```diff
@@ -14,6 +14,20 @@ ConditionPathIsDirectory=/sys/module/zfs
 [Service]
 Type=oneshot
 RemainAfterExit=yes
+EnvironmentFile=@initconfdir@/zfs
```

Comment:
```suggestion
EnvironmentFile=-@initconfdir@/zfs
```
It might be safer to assume that the file might not exist, unless explicitly created by user.

Reply:
You are right, `zfs` is shipped with OpenZFS. I think we could safely assume its existence.
I try to be cautious as different distros package in different ways.

Reply:
fixed